rest/src/db/CatapultDb.js

```diff
@@ -225,6 +225,17 @@ class CatapultDb {
 		).then(this.sanitizer.renameId);
 	}
 
+	blocksAtHeights(heights) {
+		if (!heights.length)
+			return Promise.resolve([]);
+
+		return this.queryDocumentsAndCopyIds(
+			'blocks',
+			{ 'block.height': { $in: heights } },
+			{ projection: { 'meta.transactionMerkleTree': 0, 'meta.statementMerkleTree': 0 } }
```

Comment:
why this projection?

Reply:
don't you only need timestamp?

Reply:
> why this projection?

To keep it consistent with the singular `blockAtHeight` 
https://github.com/symbol/catapult-rest/blob/e730a5ea4d8c25fc4aedf534962de4040eef3c83/rest/src/db/CatapultDb.js#L220. 

> don't you only need timestamp?

In this case yes, for #637 fix I would need the feeMultipler. 

Having said that, I'll add the custom projection.

Reply:
Added custom projection + unit tests